app/components/plan/Workout.js

```diff
@@ -29,13 +30,18 @@ export default ({ treatment, num }) => {
         {`: ${treatment.notes ? treatment.notes : 'No notes specified'}`}
       </p>
       <div className="col-xs-2 workout-button">
-        <StepsActionButton disabled={ completedWorkout ? true : false }>
-          {
-            completedWorkout ?
-              <img src={require(`../../../src/images/emojis/${completedWorkout.pain}pain.svg`)} /> :
+      {
+        completedWorkout ?
+          <StepsActionButton disabled={true}>
+            {/* TODO: Currently defaulting to pain level 3 when no input -> need real default */}
```

Comment:
We might just handle this on the server side and default to 0 when no input. Or in other words, for analytics purposes, a 0 would be treated as no user input. Another option is to assume no pain if the user skips it, so default to 1.